rero_ils/modules/acq_order_lines/api.py

```diff
@@ -217,6 +224,18 @@ def unreceived_quantity(self):
         """Get quantity of unreceived ordered_items for a line order."""
         return self.quantity - self.received_quantity
 
+    @cached_property
+    def receipt_date(self):
+        """Get the first reception date for one item of this order line."""
+        from rero_ils.modules.acq_receipt_lines.api import \
+            AcqReceiptLinesSearch
+        search = AcqReceiptLinesSearch() \
```

Comment:
Do I understand correctly, you are doing this because a `librarian` can receive some `quantity_items` in a `receipt_1` -> `receipt_line_1_1` and the rest of the `quantity_items` in another `receipt_2` -> `receipt_line_2_1`?